src/components/Button.js

```diff
@@ -17,7 +18,15 @@ class Button extends React.Component {
 }
 
 Button.propTypes = {
-  kind: React.PropTypes.oneOf(styles.buttons).isRequired
+  kind: React.PropTypes.string
+};
+
+Button.propTypes = {
+  children: React.PropTypes.string
+};
+
+Button.propTypes = {
+  style: React.PropTypes.object
```

Comment:
You want one `Button.propTypes` declaration. Here you are just rewriting the value of `Button.propTypes`. It should look something like this:
```javascript
Button.propTypes = {
  kind: React.PropTypes.string,
  children: React.PropTypes.node,
  style: React.PropTypes.object
};
```